code/ethan/html/lab01.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Doomguy</h1>
+
+    <img src="https://yt3.ggpht.com/a/AATXAJy1XH2HeynHBXyZl2Z1dx3TWX7Df5zgHf22Zg=s900-c-k-c0xffffffff-no-rj-mo">
+    
+    <a href="URL">https://en.wikipedia.org/wiki/Doomguy</a>
```

Comment:
* This <a> tag does not work properly.
    * Check out some sources for proper use of <a> tags.
        * Two examples are:
            * https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/docs/02%20HTML%20Elements.md#anchor-tags-a
            * https://www.w3schools.com/tags/tag_a.asp